src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -188,6 +188,7 @@
     private static final String ACTION_SUMMARY_TABLE_NAME = "actions";
 
     protected static final String PERMISSION_ERROR = "User does not have permission to perform this operation.";
+    protected static final String NOT_FOUND_ERROR = "notFound";
```

Comment:
Move `PERMISSION_ERROR` and `NOT_FOUND_ERROR` to `SecurityTest`. The only other place that uses `PERMISSION_ERROR` is an unused method, which can be removed: `BaseWebDriverTest.assertAtUserUserLacksPermissionPage`.